aggregator/interface.go

```diff
@@ -14,8 +14,16 @@ type PriceFetcher interface {
 	IsInterfaceNil() bool
 }
 
+// ArgsPriceChanged is the argument used when notifying the notifee instance
+type ArgsPriceChanged struct {
+	Base             string
+	Quote            string
+	Price            float64
```

Comment:
having DenominatedPrice+DenominationFactor you can compute the price value

Reply:
done. kept DenominatedPrice and changed the Price with DenominationFactor